composer.json

```
@@ -54,11 +54,10 @@
     "tao-extension-name": "generis"
   },
   "require": {
-    "php": "^7.1",
     "clearfw/clearfw": "~1.2.0",
     "easyrdf/easyrdf": "^1.1",
-    "doctrine/dbal": "^2.10.1",
-    "doctrine/annotations": "^1.6.0",
+    "doctrine/dbal": "2.12.*",
+    "doctrine/annotations": "1.13.*",
```

I strongly advise taking advantage of the semver here, and rely on the following version constraints.
```suggestion
    "doctrine/dbal": "^2.12",
    "doctrine/annotations": "^1.13",
```

This will ensure future updates while keeping the major version locked.